packages/fxa-settings/src/components/OAuthDataError/index.tsx

```diff
@@ -24,7 +24,9 @@ const OAuthDataError = ({ error }: { error: AuthError }) => {
       <Banner
         type="error"
         content={{
-          localizedHeading: getLocalizedErrorMessage(ftlMsgResolver, error),
+          localizedHeading: error.version
+            ? getLocalizedErrorMessage(ftlMsgResolver, error)
+            : error.message,
```

Comment:
There is overlap in error number between the OAuth errors and the other errors.  This is a workaround.

Reply:
I'm having flashbacks to when [Dan asked me about this](https://github.com/mozilla/fxa/pull/16750#discussion_r1575316434) 😆 Do you mind adding a `TODO in FXA-9502` comment? At least, I see I noted above the `getErrorFtlId` fn that we would account for that there. We might could use a comment on 9502 too...